_i18n/en.yml

```diff
@@ -59,6 +59,7 @@ contact_page:
       railroad_retirement_board: Railroad Retirement Board
       sam: GSA SAM.gov
       sba_dlap: SBA DLAP
+      ssa: Social Security Administration (SSA)
```

Comment:
Not introduced with these changes, but AFAICT we're not translating any agency names, which I think is expected, but also prompts the question whether to bother with including them in this file vs. hard-coding them into the template.

Reply:
yeah, I just wanted to match the existing format.

Speaking of future-facing changes, I think in an ideal world we'd generate the entire form from YAML front matter instead of having to fiddle with these inputs manually